bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/discovery/ThingDiscoveryService.java

```diff
@@ -147,8 +159,12 @@ private void addLight(String lightID, LightMessage light) {
 
         ThingUID uid = new ThingUID(thingTypeUID, bridgeUID, light.uniqueid.replaceAll("[^a-z0-9\\[\\]]", ""));
         DiscoveryResult discoveryResult = DiscoveryResultBuilder.create(uid).withBridge(bridgeUID)
-                .withLabel(light.name + " (" + light.manufacturername + ")").withProperty("id", lightID)
-                .withProperty(UNIQUE_ID, light.uniqueid).withRepresentationProperty(UNIQUE_ID).build();
+                .withLabel(light.name + " (" + light.manufacturername + ")")
+                .withProperty("id", lightID)
+                .withProperty(UNIQUE_ID, light.uniqueid)
```

Comment:
Why not put these in the properties map to begin with? It would make things consistent.